routing.md

```diff
@@ -282,23 +282,23 @@ These last two characteristics are the foundation of mix networking feature for
 
 ### Sender Forward Process
 
-1. Sender Constructs Message
-2. Sender Encrypts Message to recipient(s)
+1. Sender Constructs Message.
+2. Sender Encrypts Message to recipient(s).
 3. Wrap Encrypted Message in Forward Message for each Routing Key.
 4. Transmit to `serviceEndpoint` in the manner specified in the [transports] section.
 
 ### Mediator Process
 
 _Prior to using a Mediator, it is the recipient's responsibility to coordinate with the mediator. Part of this coordination informs them of the `to` address(es) expected, the endpoint, and any Routing Keys to be used when forwarding messages. That coordination is out of the scope of this spec._
 
-1. Receives Forward Message
-2. Retrieves Service Endpoint and Routing Keys as pre-configured by recipient (`to` attribute).
-3. Wrap attached `payload` message once per Routing Key
+1. Receives Forward Message.
+2. Retrieves Service Endpoint and (optionally) Routing Keys as pre-configured by recipient (`to` attribute).
+3. If Routing Keys pre-configured (step 2), wrap attached `payload` message once per Routing Key.
```

Comment:
This still does not match my mental model. Mediators do not commonly wrap at all. They unwrap (decrypt a `forward` message) and then forward, without rewrapping, the payload inside it. They do this behavior even if routing keys are pre-configured. The only time rewrapping would be done is if they are sending to a mediator that's not visible to the outside world.

Reply:
The ability for the recipient to arrange routing not visible to the sender is one of the useful things here. In simplest form a mediator just knows which endpoint to use for the internal payload. We could eliminate this option in favor of only the last mediator being capable of doing something like this, but I was generalizing the pattern a little. 

Are you in favor of eliminating this for the sake of simplicity?

Reply:
Not at all. I don't see why the two ideas are connected. Mediators don't have to always re-wrap to ensure this flexibility. They only re-wrap when the flexibility has been activated; they don't do it in all cases. The sender doesn't need to know which behavior they're using.

Reply:
I've updated the language of this section to extract the optional processing of additional routing keys below the process steps. Does this work for you?

Reply:
No. The fundamental misalignment in mental models is still important: mediators usually don't wrap. Let's discuss interactively.